crates/editor/src/code_context_menus.rs

```diff
@@ -722,10 +725,24 @@ impl CompletionsMenu {
                     SnippetSortOrder::Bottom => Reverse(if mat.is_snippet { 0 } else { 1 }),
                     SnippetSortOrder::Inline => Reverse(0),
                 };
+                // Fuzzy score doesn't take this criteria in count for scoring
+                // i.e. "set_collapse_matches" and "select_all" have same fuzzy score for "set"
```

Comment:
might be possible to tweak fuzzy algorithm for this

Reply:
Yeah, I'm not touching the fuzzy algorithm yet since it's used everywhere across Zed. We also don't have any tests for the core fuzzy matcher logic, so I had to resort to this hack.